sensor_msgs/Image image
int32 kCurSegment
---
int32 segmentsRows
int32 segmentsCols
int32[] segments